dataaccess/dal.go

```diff
@@ -139,3 +143,108 @@ func monitorConfig(ctx context.Context) {
 		lastConfigState = cs
 	}
 }
+
+func bootstrapUserWithDefaults(user rest.User) (rest.User, error) {
+	// If user doesn't have a defined email, we default to "gort@localhost".
+	if user.Email == "" {
+		user.Email = "gort@localhost"
+	}
+
+	// If user doesn't have a defined name, we default to "Gort Administrator".
+	if user.FullName == "" {
+		user.FullName = "Gort Administrator"
+	}
+
+	// The bootstrap user is _always_ named "admin".
+	user.Username = "admin"
+
+	// If user doesn't have a defined password, we kindly generate one.
+	if user.Password == "" {
+		password, err := data.GenerateRandomToken(32)
+		if err != nil {
+			return user, err
+		}
+		user.Password = password
+	}
+
+	return user, nil
+}
+
+func DoBootstrap(ctx context.Context, user rest.User) (rest.User, error) {
```

Comment:
I'm not sure the `Do` makes sense in this package (I'm not sure it made sense in the other one either tbh). Should it just be named `Bootstrap`? If this causes too many downstream changes it can be done later in another PR.